### PR TITLE
chore: resolve npm audit issue

### PR DESCRIPTION
**Pull request by jrschumacher**
- Fix issues with npm audit
- Fix issue with commit message linter
- Enhance commit message linter

```
-------------------------------------------------------------------------------------------
This project uses conventional commits, please make sure your commit message looks like:

    type(scope?): subject

i.e.

    chore: run tests on github
    fix(button): pass event handler correctly
    feat(table): add filter for columns

Learn more at https://www.conventionalcommits.org.
-------------------------------------------------------------------------------------------
⧗   input: .git/COMMIT_EDITMSG
✖   subject may not be empty [subject-empty]
✖   type may not be empty [type-empty]

✖   found 2 problems, 0 warnings
ⓘ   Get help: https://github.com/conventional-changelog/commitlint/#what-is-commitlint
```

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -75,37 +74,36 @@
...
-    "eslint-plugin-react-hooks": "^1.7.0",
+    "eslint-plugin-react": "^7.20.5",
+    "eslint-plugin-react-hooks": "^4.0.8",
     "identity-obj-proxy": "^3.0.0",
-    "jest": "^26.1.0",
-    "jest-dom": "^3.2.2",
-    "loki": "^0.20.3",
+    "jest": "^26.3.0",
+    "jest-dom": "^4.0.0",
+    "loki": "^0.24.0",
```
Comment (avkvirtru) [inline package.json:98]: This is a sizable bump in Loki. Did you check that VRT still successfully runs for you locally?

Comment (jrschumacher) [reply package.json:98]: I didn't tell Loki to update which is strange. I'll check it out.

File: .stylelintignore [hunk 2]
```diff
@@ -0,0 +1 @@
+lib/styles/build
```
Comment (avkvirtru) [inline .stylelintignore:1]: Were they complaining?

File: .stylelintrc.js [hunk 3]
```diff
@@ -43,6 +43,11 @@ module.exports = {
     
     // why? trailing zeros align CSS properties for comparisons across lines
     "number-no-trailing-zeros": null,
+
+    // why? composes uses css variables and should not force classnames to be lowercase
```
Comment (avkvirtru) [inline .stylelintrc.js:47]: thx for following my convention 😃 !

File: .stylelintignore (hunk 2 above)
Comment (jrschumacher) [reply .stylelintignore:1]: Yep the formatting didn't meet the spec

**Comment by jrschumacher** [comment]
:rocket: PR was released in [`v1.2.0`](https://github.com/avkvirtru/virtuoso-design-system/releases/tag/v1.2.0) :rocket:

**Pull request merged**